docs/ext/commands/commands.rst

```diff
@@ -585,6 +585,60 @@ handlers that allow us to do just that. First we decorate an error handler funct
 The first parameter of the error handler is the :class:`.Context` while the second one is an exception that is derived from
 :exc:`~ext.commands.CommandError`. A list of errors is found in the :ref:`ext_commands_api_errors` page of the documentation.
 
+
+Custom Defaults
+---------------
+
+Custom defaults allow us to specify :class:`.Context`-based defaults. Custom defaults are always classes which inherit from
+:class:`.CustomDefault`.
+
+The library provides some simple default implementations in the :module:`default` - :class:`default.Author`, :class:`default.Channel`,
+and :class:`default.Guild` returning the corresponding properties from the Context. These can be used along with Converters to
+simplify your individual commands. You can also use :class:`default.Call` to quickly wrap existing functions.
+
+A DefaultParam returning `None` is valid - if this should be an error, raise :class:`.MissingRequiredArgument`.
+
+.. code-block:: python3
```

Comment:
I feel like specific lines should be highlighted for this as it is a long codeblock.

Reply:
Btw how do you highlight them

Reply:
```rst
.. code-block:: python3
    :emphasize-lines: 1,2
        
        print('hello world!')
        print('test.')
```